doc/06-distributed-monitoring.md

```diff
@@ -3270,9 +3270,28 @@ Intermediate CA restrictions:
 
 * Each side has to provide its intermediate CAs along with the leaf certificate
   in `/var/lib/icinga2/certs/NODENAME.crt`, ordered from leaf to root.
-* Intermediate CAs may not be used directly as root CAs. To trust only specific
-  intermediate CAs, cross-sign them with themselves, so that you get equal
-  certificates except that they're self-signed. Use them as root CAs in Icinga.
+* Intermediate CAs may not be used directly as root CAs.
+
+##### Using external intermediate CA as Icinga root CA
+
+For Icinga to trust only its own CA, if the latter shall be an intermediate one,
+do either of the following, depending on how strict your company policy is:
+
+###### Lax policy, Icinga itself issues leave certificates
```

Comment:
Why is this policy lax and the other strict? This has some implications, potentially leading people to irrational decisions. I would suggest dropping the rating.

```suggestion
###### Leaf certificates issued by Icinga
```